scripts/find-debuginfo.sh

```diff
@@ -489,6 +489,7 @@ if $run_dwz \
    && [ -d "${RPM_BUILD_ROOT}/usr/lib/debug" ]; then
   readarray dwz_files < <(cd "${RPM_BUILD_ROOT}/usr/lib/debug"; find -type f -name \*.debug | LC_ALL=C sort)
   if [ ${#dwz_files[@]} -gt 0 ]; then
+    size_before="`du -s ${RPM_BUILD_ROOT}/usr/lib/debug | cut -f1`"
```

Comment:
I would use some different style of code here: `size_before=$(du -s … | cut -f1)`

Reply:
Fixed.